build/release/authors.js

```diff
@@ -0,0 +1,105 @@
+"use strict";
+
+const fs = require( "node:fs" );
+const util = require( "node:util" );
+const exec = util.promisify( require( "node:child_process" ).exec );
+const rnewline = /\r?\n/;
+const rdate = /^\[(\d+)\] /;
+
+const ignore = [
+	/dependabot\[bot\]/
+];
+
+function compareAuthors( a, b ) {
+	const aName = a.replace( rdate, "" ).replace( / <.*>/, "" );
+	const bName = b.replace( rdate, "" ).replace( / <.*>/, "" );
+	return aName === bName;
+}
+
+function uniq( arr ) {
+	const unique = [];
+	for ( const item of arr ) {
+		if ( ignore.some( re => re.test( item ) ) ) {
+			continue;
+		}
+		if ( item && !unique.find( ( e ) => compareAuthors( e, item ) ) ) {
+			unique.push( item );
+		}
+	}
+	return unique;
+}
+
+function cleanupSizzle() {
+	console.log( "Cleaning up..." );
+	return exec( "npx rimraf .sizzle" );
+}
+
+function cloneSizzle() {
+	console.log( "Cloning Sizzle..." );
+	return exec( "git clone https://github.com/jquery/sizzle .sizzle" );
+}
+
+async function getLastAuthor() {
+	const authorsTxt = await fs.promises.readFile( "AUTHORS.txt", "utf8" );
+	return authorsTxt.trim().split( rnewline ).pop();
+}
+
+async function logAuthors( preCommand ) {
+	let command = "git log --pretty=format:\"[%at] %aN <%aE>\"";
+	if ( preCommand ) {
+		command = preCommand + " && " + command;
+	}
+	const { stdout } = await exec( command );
+	return uniq( stdout.trim().split( rnewline ).reverse() );
+}
+
+async function getSizzleAuthors() {
+	await cloneSizzle();
+	const authors = await logAuthors( "cd .sizzle" );
+	await cleanupSizzle();
+	return authors;
+}
+
+function sortAuthors( a, b ) {
+	const [ , aDate ] = rdate.exec( a );
+	const [ , bDate ] = rdate.exec( b );
+	return parseInt( aDate ) - parseInt( bDate );
```

Comment:
Since these are supposed to be numbers, `Number` would be better instead of `parseInt`:
```suggestion
	return Number( aDate ) - Number( bDate );
```